--- models/User.js 2026-03-03 23:36:18.000000000 +0530
+++ models/User.js 2026-03-03 23:42:00.000000000 +0530
@@ -37,6 +37,17 @@
       default: false,
     },
 
+    // 🥘 RESTAURANT ADMIN
+    isRestaurantAdmin: {
+      type: Boolean,
+      default: false,
+    },
+    adminStatus: {
+      type: String,
+      enum: ["Pending", "Approved", "Rejected"],
+      default: "Approved",
+    },
+
     // 🔥 ADD THESE TWO FIELDS
     resetToken: String,
     resetTokenExpiry: Date,
